deadfish.fbs

```diff
@@ -7,16 +7,39 @@ struct Vec2 {
   y:float;
 }
 
-table Stone {
-    radius: float;
+table Tileinfo {
+  gid: uint16;
+  name: string;
+}
+
+table Object {
+    pos: Vec2;
     rotation: float;
+    gid: uint16;
+    hspot: bool;
+    hspotname: string;
+}
+
+table Decoration {
     pos: Vec2;
+    rotation: float;
+    gid: uint16;
 }
 
-table HidingSpot {
+table Collision {

```

Comment:
The name is slightly misleading, as "Collision" doesn't mean the event of collision but rather "something that can be collided with". What do you think about renaming this to "Obstacle" or something like that?

Reply:
Done. Renamed to CollisionMask everywhere